content/webapp/components/IIIFViewer/GridViewer.tsx

```diff
@@ -117,6 +117,7 @@ const Cell = memo(({ columnIndex, rowIndex, style, data }: CellProps) => {
 
 Cell.displayName = 'Cell';
 
+// TODO these seem unused; should they be removed?
```

Comment:
Opinions? I started doing it but it felt like a lot to remove for an unrelated PR

Reply:
let's bin them

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/issues/10381